Source/Resources/Content/MSGraph/Groups/BUILTIN - Corporate Devices - Preview.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "Used to assign preview policies and applications",
+  "description": "What is it: A manual device group for managing preview policies and applications. \n Simeon Notes: A good use case for this group is to test application installs and compatibility with a subset of devices in your environment. Devices in this group will be assigned to the Preview Windows Update ring.",
```

Comment:
**What does this do?**
Creates a manual device group for previewing and testing policies and applications. Devices in this group will be assigned to the Preview Windows Update ring.

**Why should you use this?**
If you want to test application installs and compatibility with a subset of devices in your environment. 

**What is the end user impact?**
Users of this group will be able to test application installs and policies separately from other users.

**Learn more**
NA